### PR TITLE
OPENEUROPA-1322: Use the proper Docker way of overriding docker-composer.yml file. 

### PR DESCRIPTION
**Pull request by drupol**
OPENEUROPA-1322: Use the proper Docker way of overriding docker-composer.yml file.

### CODE REVIEW

**Review by richardcanoe — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -143,6 +165,14 @@ parameters:
 
 This is due to the following [Drupal Console issue][11].
 
+## Contributing
+
+Please read [the full documentation](https://github.com/openeuropa/openeuropa) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the available versions, see the [tags on this repository](https://github.com/openeuropa/drupal_module_template/tags).
```
Comment (dxvargas) [inline README.md]: Wrong link, it must refer to media_avportal.

File: README.md [hunk 2]
```diff
@@ -56,7 +57,7 @@ to `./runner.yml` and overriding relevant properties.
 To set up the project run:
 
 ```bash
-$ ./vendor/bin/run drupal:site-setup
+./vendor/bin/run drupal:site-setup
```
Comment (dxvargas) [inline README.md]: This should be removed like in other repositories we are now updating.

**Review by dxvargas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (drupol) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (drupol) [reply README.md]: Fixed.

**Review by dxvargas — APPROVED** [review]

**Review by dxvargas — APPROVED** [review]

**Review by richardcanoe — APPROVED** [review]

**Pull request merged**